types/eureka.go

```diff
@@ -0,0 +1,16 @@
+package types
+
+// EurekaPayload defines a single packet sent in the EurekaSendPacketMsg
+//
+// Payload value should be encoded in a format defined by the channel version,
```

Comment:
Do you have a source for this? The channel is now the connection between two chains, so why is the channel version relevant for the contract? Or do you mean the `Version` field in this struct?

Reply:
I meant the `Encoding` field. Have to check the std doc comment too. Good catch!

Reply:
Yup, same fix needs to be applied in cosmwasm-std. Could you create a PR?